crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/stateless/StatelessAssessmentService.java

```diff
@@ -34,97 +31,92 @@ public StatelessAssessmentService(AssessmentCriteriaService assessmentCriteriaSe
+                .assessmentStatus(CurrentStatus.COMPLETE)
+                .childWeightings(children)
+                .initTotalAggregatedIncome(totalIncome)
+                .eligibilityCheckRequired(eligibilityCheckRequired)
+                .build();
 
-    private StatelessResult initialOnly(@NotNull Assessment assessment,
-                                        @NotNull Map<AgeRange, Integer> childGroupings,
-                                        @NotNull List<Income> incomes) {
+        final var totalOutgoings = calcOutgoingTotals(criteriaEntry, caseType, outgoings);
+        final var fullAssessmentService = meansAssessmentServiceFactory.getService(AssessmentType.FULL);
+        final var fullAssessmentResult = fullAssessmentService.execute(totalOutgoings, requestDTO, criteriaEntry);
 
-        final var criteriaEntry = assessmentCriteriaService.getAssessmentCriteria(
-                assessment.getAssessmentDate(), assessment.getHasPartner(), false
-        );
-        final var totalIncome = calcIncomeTotals(criteriaEntry, assessment.getCaseType(), incomes);
-        var initialAssessment = initialResult(assessment, totalIncome, childGroupings);
-        return new StatelessResult(null, initialAssessment);
+        return StatelessFullResult.builder()
+                .result(fullAssessmentResult.getFullAssessmentResult())
+                .disposableIncome(fullAssessmentResult.getTotalAnnualDisposableIncome())
+                .adjustedLivingAllowance(fullAssessmentResult.getAdjustedLivingAllowance())
+                .totalAnnualAggregatedExpenditure(fullAssessmentResult.getTotalAggregatedExpense())
+                .eligibilityThreshold(criteriaEntry.getEligibilityThreshold()).build();
     }
 
-    private StatelessInitialResult initialResult(@NotNull Assessment assessment,
-                                                 @NotNull BigDecimal totalIncome,
-                                                 @NotNull Map<AgeRange, Integer> childGroupings) {
+    public StatelessInitialResult initialResult(@NotNull Map<AgeRange, Integer> childGroupings,
+                                                 AssessmentCriteriaEntity criteriaEntry,
+                                                 CaseType caseType,
+                                                 MagCourtOutcome magCourtOutcome,
+                                                 List<Income> incomes,
+                                                 NewWorkReason newWorkReason) {
 
-        final var criteriaEntry = assessmentCriteriaService.getAssessmentCriteria(
-                assessment.getAssessmentDate(), assessment.getHasPartner(), false
-        );
         final var children = mapChildGroupings(childGroupings, criteriaEntry.getAssessmentCriteriaChildWeightings());
 
+        final var totalIncome = calcIncomeTotals(criteriaEntry, caseType, incomes);
+        final var service = meansAssessmentServiceFactory.getService(AssessmentType.INIT);
```

Comment:
There are lots of services - how about we name this variable explicitly `initMeansAssessmentService`?